cmake/PGI.cmake

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Is anyone going to test this? I won't.

Reply:
Probably not. Should we remove all references to PGI?

Reply:
I just read through the platform and compiler support policy. PGI is listed as Tier 3, and per definition for Tier 3 compiler option template must be provided (albeit compilation and execution are not guaranteed). So, let's keep it untested ...